### PR TITLE
Test proj

### PR DESCRIPTION
**Pull request by awirb**

### CODE REVIEW

File: avaframe/com1DFA/com1DFACfg.ini [hunk 1]
```diff
@@ -28,3 +26,14 @@ aimecDir = Outputs/com1DFA/
 calcPressureLimit = 10
 # width of the avalanche path domain for computations in Aimec
 domainWidth = 600
```
Comment (matthiasto) [inline avaframe/com1DFA/com1DFACfg.ini:28]: This is not needed anymore. It is set in the AIMEC.ini file
You don't even need to write this .txt file anymore

File: avaframe/com1DFA/com1DFACfg.ini [hunk 2]
```diff
@@ -28,3 +26,14 @@ aimecDir = Outputs/com1DFA/
...
+
+
+[PARAMATERVAR]
+# True if parameter variation of MU option shall be included
+flagVarPar = False
+# parameter to be varied, current option: Mu (friction parameter)
+varPar = Mu
```
Comment (matthiasto) [inline avaframe/com1DFA/com1DFACfg.ini:35]: can we change the Mu to something like depth? will it work?

**Review by matthiasto — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/com1DFACfg.ini (hunk 1 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFACfg.ini:28]: Ok, now there is no section Aimec anymore in the .ini

File: avaframe/com1DFA/com1DFACfg.ini (hunk 2 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFACfg.ini:35]: I added the option to change the release area thickness

**Pull request merged**